PubSub/tests/Snippet/MessageTest.php

```diff
@@ -142,6 +143,15 @@ public function testAckId()
         $this->assertEquals($this->metadata['ackId'], $res->output());
     }
 
+    public function testDeliveryAttempt()
```

Comment:
Please add a test for when deliveryAttempt is not set.

Reply:
added a unit test.